js_course/test-1.spec.js

```diff
@@ -0,0 +1,14 @@
+import { test, expect } from '@playwright/test';
+
+test('test-1', async ({ page }) => {
+  await page.goto('https://www.saucedemo.com/');
+  await page.locator('[data-test="username"]').click();
```

Comment:
Please, check this doc https://playwright.dev/docs/api/class-page#page-get-by-test-id

```
// Set custom test id attribute from @playwright/test config:
import { defineConfig } from '@playwright/test';

export default defineConfig({
  use: {
    testIdAttribute: 'data-pw'
  },
});
```

Then the locator will be easier to read.

```
await page.getByTestId('username').click();
```